pkg/mountkexec/mountkexec.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package webboot
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/u-root/u-root/pkg/loop"
+	"github.com/u-root/u-root/pkg/mount"
+	"golang.org/x/sys/unix"
+)
+
+// MountISO runs a command with mount,unix and loop
+func MountISO(isoPath, mountDir string) error {
+	if err := os.MkdirAll(mountDir, 777); err != nil {
+		return fmt.Errorf("error making mount directory:%v", err)
+	}
+	loopDevice, err := loop.FindDevice()
+	if err != nil {
+		return fmt.Errorf("error finding loop device:%v", err)
+	}
+	if err := loop.SetFile(loopDevice, isoPath); err != nil {
+		return fmt.Errorf("error setting loop device:%v", err)
+	}
+	var flags uintptr
+	flags |= unix.MS_RDONLY
+	if err = mount.Mount(loopDevice, mountDir, "vfat", "", flags); err != nil {
```

Comment:
vfat?

Reply:
This is the formart I am testing with for now.I will change it once I get an ISO